uber/decorators.py

```diff
@@ -279,8 +274,72 @@ def __call__(self, klass):
 
 
 register = template.Library()
+
+
 def tag(klass):
     @register.tag(klass.__name__)
     def tagged(parser, token):
         return klass(*token.split_contents()[1:])
     return klass
+
+
```

Comment:
This is our new approach for validations.  Instead of "functions in ``model_checks.py`` are used as validators" we now have a decorator, so you can say

```python

```

I still need to write some more docstrings for this.